crates/bevy_ecs/src/world/identifier.rs

```diff
@@ -27,6 +30,13 @@ impl WorldId {
     }
 }
 
+impl FromWorld for WorldId {
+    #[inline]
+    fn from_world(world: &mut World) -> Self {
+        world.id()
```

Comment:
This property is subtle and important. Can you add a note about it to the docs for `WorldId`?

Reply:
What do you think needs to be noted?

Reply:
The fact that this fetches the id of the world that it belongs to, rather than an arbitrary or default ID.